.github/workflows/robustness.yaml

```diff
@@ -3,16 +3,6 @@ name: Robustness
 on: [push, pull_request]
```

Comment:
Please remove both presubmits and non-arm nightly robustness test.

No benefit of having presubmit on arm64.

Reply:
Sure 👍